tests/end_to_end/test_annotate_pdf.py

```diff
@@ -249,6 +249,13 @@ def _add_text_annotations(self, a, y1=220, y2=300):
                 Location(x1=x, y1=y1, x2=(x + 40), y2=y2, page=0),
                 appearance,
             )
+            # TODO: Proper way to enable?
+            # a.add_annotation(
```

Comment:
This looks right. You'd have to run the test and then actually open the PDF and make sure the annotation appears as expected.

Reply:
Ok great! I'll test it out and will post an update.

Reply:
hi has there been any updates on this PR?